geomstats/visualization/discrete_curves.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python
```

Comment:
Format the information at the top of this file: put it in a docstring format, as done in other .py files in geomstats.